exampleplugin/example.go

```diff
@@ -0,0 +1,23 @@
+package main
+
+import (
+	"github.com/opencontrol/compliance-masonry/lib"
+	"fmt"
+)
+
+type plugin struct {
+	lib.Workspace
+}
+
+func simpleDataExtractAndFormat(p plugin) string {
+	selectJustifications := p.GetJustification("standard", "control")
+	if len(selectJustifications) == 0 {
+		return "no data"
+	}
+	return selectJustifications[0].SatisfiesData.GetImplementationStatus()
+}
+
+func main() {
+	workspace, _ := lib.LoadData("sample opencontrol directory", "cert path")
+	fmt.Println(simpleDataExtractAndFormat(plugin{workspace}))
```

Comment:
Minor: I'd split this to two lines so it's clearer what the return value of `simpleDataExtractAndFormat()` is.